packages/wrangler/scripts/bundle.ts

```diff
@@ -48,7 +48,7 @@ async function buildMiniflareCLI(flags: BuildFlags = {}) {
 		outfile: "./miniflare-dist/index.mjs",
 		platform: "node",
 		format: "esm",
-		external: ["miniflare", "@miniflare/core"],
+		external: ["miniflare", "@miniflare/core", "chokidar"],
```

Comment:
could you add this to scripts/deps.ts to `EXTERNAL_DEPENDENCIES` as well? Honestly, maybe we should just use that array here